Rasa_Bot/actions/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# Dockerfile for rasa actions server
```

Comment:
This dockerfile is not new, it is just the action server Dockerfile that was previously in the Rasa_Bot root folder